var/spack/repos/builtin/packages/py-matplotlib/package.py

```diff
@@ -68,6 +68,7 @@ class PyMatplotlib(PythonPackage):
 
     depends_on('libpng@1.2:')
     depends_on('freetype@2.3:')
+    patch('freetype-include-path.patch', when='@2.2.2:') # Patch to pick up correct freetype headers
```

Comment:
Since this has been merged upstream, it should probably be `when='@2.2.2'`, unless you can get it to apply to older versions as well.